thunder/tests/opinfos.py

```diff
@@ -6018,6 +6019,56 @@ def topk_error_generator(op, device, **kwargs):
 reduction_ops.append(topk_opinfo)
 
 
+def atleast_1d2d3d_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = (
+        (),
+        (4,),
+        (5, 5),
+        (6, 7, 8),
+        (3, 3, 3, 3),
+    )
+
+    for c in cases:
+        yield SampleInput(make(c))
+
+    yield SampleInput(make(()), make((2,)))
+    yield SampleInput(make((2,)), make((5, 5)))
+    yield SampleInput(make(()), make((2,)), make((4, 4)))
+    yield SampleInput(make(2, 3), make(4, 5), make(6, 6, 6), make(5, 5, 5, 5))
+
+
+atleast_1d_opinfo = OpInfo(
+    clang.atleast_1d,
+    supports_grad=True,
+    dtypes=(datatypes.floating, datatypes.exact, datatypes.complexfloating),
```

Comment:
Isn't this the default value for dtypes? I think this line can be deleted, here and in the other opinfos.